docs/page/main-features.md

```diff
@@ -94,6 +94,20 @@ In general the formatter settings are language server specific(e. g. `JDT LS` us
 
 ![](../examples/lsp-dart-flutter-debug.gif)
 
+## iedit
+
+`lsp-mode` can leverage [`iedit`](https://github.com/victorhge/iedit) to edit
+semantic matches in parallel:
+
+- `lsp-iedit-highlights`: invoke `iedit` on the symbol highlights at point, all
+  of which will be edited in parallel.
+  
+  To finish and exit, press `C-g`, with the cursor on `iedit` highlight.
+  
+  If an `iedit` session is already active, `lsp-iedit-highlights` will simply
+  add the document highlights to it, without restarting it. This way, another
+  group of symbol highlights can be added to the current `iedit` session.
+
```

Comment:
I think this would fit better as a integration like Helm, ivy etc, check the above section: `Integrations`

Reply:
I agree. How would I go about doing that? The other `Extensions: ` target separate projects with different github.io pages and I am unfamiliar with `mkdocs`. Could you point me in the right direction?

Reply:
You can add a entry like this:
```yaml
    - Extensions:
        - UI: https://emacs-lsp.github.io/lsp-ui
        - Treemacs: https://github.com/emacs-lsp/lsp-treemacs
        - Helm: https://github.com/emacs-lsp/helm-lsp
        - Ivy: https://github.com/emacs-lsp/lsp-ivy
        - Iedit: page/integrations/iedit.md
```
Also, I realized we are missing avy integration too 😅 or mentioning in other hidden place

Reply:
Thank you, I've implemented your suggestion. However, I still don't have a way to preview them, as `make local-webpage` fails:
```
[...]
docker run --rm -it -p 8000:8000 -v /home/nikita/Projects/emacs-package-contrib/lsp-mode:/docs docker.pkg.github.com/emacs-lsp/docs-image/docs-image
Unable to find image 'docker.pkg.github.com/emacs-lsp/docs-image/docs-image:latest' locally
docker: Error response from daemon: unauthorized: Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured.
See 'docker run --help'.
make: *** [Makefile:99: local-webpage] Error 125
```

Do you know what might be wrong? Why would it need to authenticate itself with github? My system is Ubuntu 20.04 LTS, I have docker installed and my user is part of the `docker` group.